lib/transport/selenium-webdriver/method-mappings.js

```diff
@@ -590,9 +590,17 @@ module.exports = class MethodMappings {
           return null;
         },
 
-        setElementValueRedacted(...args) {
-          // FIXME: redact password in verbose HTTP logs, it's only redacted in the command logs
-          return this.methods.session.setElementValue.call(this, ...args);
+        setElementValueRedacted(webElementOrId, value) {
+          let redactedValue;
+          if (value instanceof Array) {
+            value.unshift(Key.NULL);
+            redactedValue = value;
+          } else {
+            // Can be String or number.
+            redactedValue = Key.NULL + value;
+          }
```

Comment:
```suggestion
          // prepend `NULL` to value to distinguish it later on and redact it from logs
          const modifiedValue = [Key.NULL].push(Array.isArray(value) ? ...value : value);
```

Reply:
found a better way that can let us circumvent the check.
Array.concat(). 
```
const modifiedValue = [Key.NULL].concat(value);

return this.methods.session.setElementValue.call(this, webElementOrId, modifiedValue);
```

Reply:
Cool